tap_pipedrive/tap.py

```diff
@@ -7,6 +7,7 @@
 from singer import set_currently_syncing, metadata
 from singer.catalog import Catalog, CatalogEntry, Schema
 from .config import BASE_URL, CONFIG_DEFAULTS
```

Comment:
See next comment
```suggestion

```

Reply:
Darn, I thought GitHub would keep this above the other comment. So by "See next comment",  I mean see the comment about line 10